src/note.jl

```diff
@@ -180,7 +181,45 @@ function name_to_pitch(name)
     
     return pitch + x + 12(octave+1) # lowest possible octave is -1 but pitch starts from 0
 end
+"""
+    pitch_to_hz(name::String, a4::Real) -> Float
+Return the frequency value of the given midi note , which can be of the form
+`Real` where:
 
+We define E.g. `midi_to_hz(69) === 440(with the default middle a)` 
+See https://en.wikipedia.org/wiki/Piano_key_frequencies
+and https://librosa.org/doc/main/_modules/librosa/core/convert.html#midi_to_hz.
+"""
+function pitch_to_hz(pitch, A4 = 440)
+    return A4 * (2^ ((pitch-69) / 12))
+end
+"""
+    hz_to_pitch(name::String,A4::Real) -> Real
+Return the midi value of the given  frequency
+default middle a is 440hz
+"""
+function hz_to_pitch(freq, A4 = 440)
+    return 12 * (log2(freq) - log2(A4)) + 69
+end
+"""
+hz_to_name(freq::Real,A4::Real) -> String
```

Comment:
```suggestion
    hz_to_name(freq::Real, A4::Real = 440) -> name::String
```
Yoou need 4 white spaces for it to register as code. also, it is useful to add the defaults to the docstring.

Reply:
👍